app/test_main.py

```diff
@@ -1,3 +1,16 @@
 from app.main import get_human_age
 
-# write your code here
+def test_should_return_zero_if_cat_and_dog_age_equal_14():
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_should_return_extra_human_year_after_24_animal_years():
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_should_return_5_human_years_if_cat_years_equals_36():
+    assert get_human_age(36, 28) == [5, 2]
```

Comment:
Ensure that your tests cover edge cases, such as negative numbers, zero, or very large numbers, as mentioned in the checklist. This will help verify that the function handles unexpected input gracefully.